### PR TITLE
Remove executed runtime upgrade code

### PR DESCRIPTION
**Pull request by adamdossa**
This PR removed runtime upgrade code which has already been executed on Alcyone 2.1.

See `alcyone` branch for storage which has already been updated.

Also fixes storage typo in:  
```
migrate_map::<Vec<TrustedIssuerOld>, _>(b"ComplianceManager", b"TrustedClaim**Isuer**", |_| Empty);
```

A companion PR will be made for the `tooling_v2.2.0` branch.

### CODE REVIEW

**Review by satyamakgec — APPROVED** [review]

**Comment by satyamakgec** [comment]
@adamdossa FYI We can also remove the settlement pallet upgrade as well.

**Comment by maxsam4** [comment]
@satyamakgec iirc more changes have been made to the Settlement module's storage after the alcyone cut. We'll need to add migration for those if we decide to remove the current migration that resets the storage.

**Review by maxsam4 — APPROVED** [review]

File: pallets/compliance-manager/src/lib.rs [hunk 1]
```diff
@@ -117,17 +117,12 @@ pub trait Trait:
     type MaxConditionComplexity: Get<u32>;
 }
 
-use polymesh_primitives::condition::ConditionOld;
-
 /// A compliance requirement.
 /// All sender and receiver conditions of the same compliance requirement must be true in order to execute the transfer.
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(Encode, Decode, Default, Clone, PartialEq, Eq, Debug, Migrate)]
```
Comment (Centril) [inline pallets/compliance-manager/src/lib.rs:123]: You still have `#[derive(Migrate)]` here.

File: pallets/compliance-manager/src/lib.rs [hunk 2]
```diff
@@ -200,12 +195,10 @@ impl From<Condition> for ConditionResult {
 /// List of compliance requirements associated to an asset.
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
 #[derive(Encode, Decode, Default, Clone, PartialEq, Eq, Migrate)]
```
Comment (Centril) [inline pallets/compliance-manager/src/lib.rs:197]: Same here.

File: primitives/src/condition.rs [hunk 3]
```diff
@@ -121,10 +119,8 @@ impl Migrate for TrustedIssuerOld {
 /// Type of claim requirements that a condition can have
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, Migrate)]
-#[migrate_context(Option<crate::CddId>)]
 pub struct Condition {
     /// Type of condition.
-    #[migrate]
     pub condition_type: ConditionType,
     /// Trusted issuers.
     #[migrate(TrustedIssuer)]
```
Comment (Centril) [inline primitives/src/condition.rs:126]: Was it intentional not to remove the `migrate` here?

Comment (adamdossa) [reply primitives/src/condition.rs:126]: Yes - TrustedIssuers still need to be migrated as part of Alcyone 2.2 > 2.3.

**Pull request merged**